jans-cedarling/cedarling_pg/scripts/install.sh

```
@@ -0,0 +1,289 @@
+    install_file "${lib_dir}/${EXT_NAME}.so" "${pkg_lib}/${EXT_NAME}.so"
+
+    shopt -s nullglob
+    local f
+    for f in "${share_dir}/${EXT_NAME}.control" "${share_dir}/${EXT_NAME}"--*.sql; do
+        log "Installing $(basename "${f}")"
+        install_file "${f}" "${ext_share}/$(basename "${f}")"
+    done
+    shopt -u nullglob
+
+    local sql_count
+    sql_count="$(find "${share_dir}" -maxdepth 1 -name "${EXT_NAME}"--*.sql | wc -l)"
```

Shellcheck (0.11.0)

[warning] Quote the parameter to -name so the shell won't interpret it.

(SC2061)